source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -383,6 +389,60 @@ namespace
         printAssistedCapture3DResults(allDurations);
     }
 
+    
+    template<typename DataType>
+    void CopyDataTime(Zivid::PointCloud pointCloud, DataType copyname)
+    {
+        std::vector<Duration> copyDataDurations;
+
+        const auto beforecopyData = HighResClock::now();
+        const auto tempcopy = pointCloud.copyData<decltype(copyname)>();
```

Comment:
You don't need to save it to a variable, this is enough:

`pointCloud.copyData<decltype(copyname)>();`